src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +17,13 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .flatMap((s) -> Arrays.stream(s.split(",")))
```

Comment:
All magic numbers, strings, and symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)

Never use 1-2 letter abbreviations for naming, even in steams and lambdas.
In a week when you forget code context, you will hate such a decision.
Please recheck all methods
```suggestion
                .flatMap(lineNumbers -> Arrays.stream(lineNumbers.split(SEPARATOR)))
```